Modular_arithmetic/include/CGAL/Modular_arithmetic/Residue_type.h

```diff
@@ -59,17 +59,13 @@ class Residue:
   typedef Residue NT;
   
 private:
-#ifdef CGAL_HEADER_ONLY
+
   static const double& get_static_CST_CUT()
   {
     static const double CST_CUT = std::ldexp( 3., 51 );
     return CST_CUT;
   }
-#else // CGAL_HEADER_ONLY
-  CGAL_EXPORT static const double  CST_CUT; 
-  static const double& get_static_CST_CUT()
-  { return Residue::CST_CUT; }
-#endif // CGAL_HEADER_ONLY
+
```

Comment:
The initialization of `CST_CUT` is not thread-safe, but I doubt a race-condition could have any visible effect: in my opinion at worst, the variable is initialized twice with the same value.